finality-aleph/src/sync/metrics.rs

```diff
@@ -68,8 +72,10 @@ const ERRORING_EVENTS: [Event; 9] = [
 
 pub enum Metrics {
     Prometheus {
-        calls: HashMap<Event, Counter<U64>>,
-        errors: HashMap<Event, Counter<U64>>,
+        event_calls: HashMap<Event, Counter<U64>>,
+        event_errors: HashMap<Event, Counter<U64>>,
+        top_finalized_block: Counter<U64>,
```

Comment:
if you would change it to gauge it would eliminate the need to do some math for the inc_by. Anyway you are checking in the update fn if the number is increasing so counter IMHO is not needed

Reply:
Hmm, using Gauge is easier, but the value of best finalized block doesn't decrease under normal conditions (I think even when sth went terribly wrong substrate cannot revert finalized blocks), so semantically it should be a counter.

Reply:
yep, exactly. It will never decrease (and if it will well we will have other  problems) so we can use gauge and not worry about this property

Reply:
ok changed